packages/nimble-components/src/theme-provider/tests/theme-provider.stories.ts

```diff
@@ -0,0 +1,50 @@
+import { html } from '@microsoft/fast-element';
+import type { Meta, StoryObj } from '@storybook/html';
+import { createUserSelectedThemeStory } from '../../utilities/tests/storybook';
+import { hiddenWrapper } from '../../utilities/tests/hidden';
+
+const metadata: Meta = {
+    title: 'Internal/Theme Provider',
+    parameters: {
+        docs: {
+            description: {
+                component: ''
+            }
+        }
+    }
+};
+
+export default metadata;
+
+const langDescription = `Defines the language of the element. See [external documentation](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/lang) for details.
+
+Users should set \`lang\` on the root \`html\` element of the page to reflect the language of the content. If necessary, users may override the language for a subtree by inserting a \`nimble-theme-provider\` element and setting its \`lang\` attribute. Nimble elements will not honor a \`lang\` value set on any other type of ancestor element.`;
+
+export const themeProvider: StoryObj = {
+    render: createUserSelectedThemeStory(hiddenWrapper(html`<style></style>`)),
+    argTypes: {
+        theme: {
+            description:
+                'The display theme to use. One of `Theme.light`, `Theme.dark`, or `Theme.color`.',
```

Comment:
We should do the attribute values. That's what we do for other enum style attributes and what users would use in Angular templates

Reply:
I know that's our convention for the values listed by radio buttons, but I thought it would be better to document the `Theme` members, as that's what we want clients to use. But I don't feel strongly, so I changed it for `lang` and for `direction`. I also realized that the `Direction` enum is a FAST type, and we weren't re-exporting it. So I did that, too.